frontend/src/views/web/change-password.html

```diff
@@ -40,7 +40,7 @@
                         <div class="align-left reg-control">
                             <button class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14" type="submit" value="Submit">Change Password</button>
                         </div>
-                        <div ng-if="changepwd.isFormError" class="wrn-msg text-highlight align-left"> {{changepwd.FormError}}</div>
+                        <div ng-if="changepwd.isFormError" class="wrn-msg text-highlight align-right"> {{changepwd.FormError}}</div>
```

Comment:
Don't align right. If error text is too large then some of part of it will hide again. 
For now remove `align-right` and  add a `</br>` after line 42.